GitUI/GitUI.csproj

```diff
@@ -65,6 +66,9 @@
     <PackageReference Include="Microsoft-WindowsAPICodePack-Shell">
       <Version>1.1.3.3</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel">
```

Comment:
I would recommend AppInsights.WindowsDesktop instead as the persistence channel is more robust. It writes to disk first and resends on next launch if needed.

Reply:
Thank you for taking time to review 👍